sopel/irc/capabilities.py

```diff
@@ -1,4 +1,21 @@
-"""Management of IRC server capabilities."""
+"""Tracking of IRC's capability negotiation.
```

Comment:
```suggestion
"""Management of IRC capability negotiation.
```

Splitting the difference between old and new. 😁